templates/vue/src/components/SettingsModal.vue

```diff
@@ -192,11 +247,19 @@ export default {
       superuserOverridePermissions: true,
       defaultDepth: 3,
       renderImages: true,
+      renderMap: false,
+      mapBounds: { neLat: 90, neLng: 180, swLat: -90, swLng: -180 },
     }
   },
   computed: {
     ...mapGetters(["tapestryJson"]),
     ...mapState(["settings", "rootId"]),
+    latbound() {
+      return (this.mapBounds.neLat || 90) > (this.mapBounds.swLat || -90)
+    },
+    lngbound() {
+      return (this.mapBounds.neLng || 180) > (this.mapBounds.swLng || -180)
+    },
```

Comment:
`latbound` and `lngbound` have 2 problems in their naming:
1. `lat` and `bound` are 2 different words and you should be using camelCase, as previously mentioned.
2. `latbound` is not explaining exactly what this variable does. You should name it something that is easy to understand like `latitudeRangeInvalid` or something like that. This will help other devs understand better what that variable is without having to look at its implementation.